# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../glibc/no-scsi-sysdeps.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Problem: scsi/sg.h scsi/scsi.h scsi/scsi_ioctl.h are header files
	 included within linux and shouldn't be included in glibc

Christian Wiese <cw@ixplanet.de> 

--- glibc-20060605/sysdeps/unix/sysv/linux/Makefile.orig	2006-09-20 23:47:15.000000000 +0200
+++ glibc-20060605/sysdeps/unix/sysv/linux/Makefile	2006-09-20 23:47:46.000000000 +0200
@@ -21,8 +21,7 @@
 		  sys/klog.h sys/kdaemon.h \
 		  sys/user.h sys/procfs.h sys/prctl.h \
 		  sys/kd.h sys/soundcard.h sys/vt.h \
-		  sys/quota.h sys/fsuid.h \
-		  scsi/sg.h scsi/scsi.h scsi/scsi_ioctl.h sys/pci.h \
+		  sys/quota.h sys/fsuid.h sys/pci.h \
 		  sys/ultrasound.h sys/raw.h sys/personality.h sys/epoll.h \
 		  bits/a.out.h sys/inotify.h
 
